tests/test_parser.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import Mock
+
+from dissect.cstruct.parser import TokenParser
+
+
+def test_preserve_comment_newlines():
```

Comment:
Maybe include the other comment options (since they are included in the pattern) also in the test and add the situations `"foo 'bar'"` and `'foo "bar"'`.

Reply:
That's not what you think it is. That regex is to leave "comments" that exist within a string alone. So e.g. `// comment` gets deleted but `"oh wowie // is such a cool operator"` doesn't.

Reply:
Aah okay, never mind then!